Building without --feature tui fails

```
...
   Compiling dav-server v0.7.0
   Compiling rustic_core v0.3.0
   Compiling rustic_backend v0.2.0
   Compiling rustic-rs v0.8.0 (/build/rustic/src/rustic-0.8.0)
error[E0432]: unresolved import `super::tui`
  --> src/commands/snapshots.rs:20:5
   |
20 | use super::tui;
   |     ^^^^^^^^^^ no `tui` in `commands`
   |
note: found an item that was configured out
  --> src/commands.rs:27:16
   |
27 | pub(crate) mod tui;
   |                ^^^
   = note: the item is gated behind the `tui` feature
```

If something is gated behind a feature, other bits of code shouldn't depend on it. The cfg feature attribute probably needs to be added more places.

--- a/src/commands/snapshots.rs
+++ b/src/commands/snapshots.rs
@@ -17,6 +17,7 @@ use rustic_core::{
     SnapshotGroupCriterion,
 };
 
+#[cfg(feature = "tui")]
 use super::tui;
 
 /// `snapshot` subcommand
@@ -47,6 +48,7 @@ pub(crate) struct SnapshotCmd {
     #[clap(long, conflicts_with_all = &["long", "json"])]
     all: bool,
 
+    #[cfg(feature = "tui")]
     /// Run in interactive UI mode
     #[clap(long, short)]
     pub interactive: bool,
@@ -63,6 +65,7 @@ impl Runnable for SnapshotCmd {
 
 impl SnapshotCmd {
     fn inner_run(&self) -> Result<()> {
+        #[cfg(feature = "tui")]
         if self.interactive {
             return tui::run(self.group_by);
         }

